dv-export/python/query-changes/data_store.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import os
+from typing import Any
+
+from sqlalchemy import String, Float, Integer, BigInteger, Boolean, insert, create_engine, Column, Table, MetaData
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy_utils import database_exists, create_database
+
+from constants import *
+
+logger = logging.getLogger(__name__)
+
+
+class DataStore:
+    """Simple data store for saving result tables to a database."""
+
+    VISIER_TO_SQL_TYPES = {
+        'String': String,
+        'Date': BigInteger,
+        'Number': Float,
+        'Integer': Integer,
+        'Boolean': Boolean
+    }
+    SQLITE_PREFIX = 'sqlite:///'
```

Comment:
Since we're using SQLAlchemy, the code should be as general as possible. 

I.e. it's fine to set the prefix to `'sqlite:///'` (though it should probably be in `constants.py`?), however the name of the symbol should be general like `DB_PREFIX`

Reply:
This code creates a directory if it doesn’t exist for SQLite. Now, I think it’s not a great idea to create it initially, so I removed it entirely.